src/components/subscription/SubscriptionItem.tsx

```diff
@@ -0,0 +1,82 @@
+import { SUBSCRIPTION_ITEM_STYLE } from '@/constants/styles';
+import type { Platform } from '@/types/recording';
+import { MinusIcon } from 'lucide-react';
+import { useState } from 'react';
+import ConfirmModal from '../common/ConfirmModal';
+import AlertModal from '../common/AlertModal';
+import useUnsubscribeMutation from '@/hooks/useUnsubscribeMutation';
+
+type SubscriptionItemProps = {
+  subscriptionId: number;
+  subscribedAt: string | null;
+  channel: {
+    channelId: string;
+    channelName: string;
+    platform: Platform;
+  };
+};
+
+const SubscriptionItem = (props: SubscriptionItemProps) => {
+  const { subscriptionId, subscribedAt, channel } = props;
+  const [alertMessage, setAlertMessage] = useState('');
+  const [showConfirmModal, setShowConfirmModal] = useState(false);
+
+  const confirmMessage = `[${channel.channelName}] 채널 구독을 해지하시겠습니까?`;
+
+  const { mutate: mutateUnsubscribe } = useUnsubscribeMutation(
+    () => setAlertMessage('채널 구독이 해지되었습니다.'),
+    () => setAlertMessage('구독채널 해지 도중 문제가 발생했습니다.')
+  );
+
+  const handleClickButton = () => {
+    setShowConfirmModal(true);
+  };
+
+  const handleConfirm = () => {
+    setShowConfirmModal(false);
+    mutateUnsubscribe(subscriptionId);
+  };
+
+  return (
+    <>
+      <div className={SUBSCRIPTION_ITEM_STYLE.container}>
+        <div className={SUBSCRIPTION_ITEM_STYLE.box}>
+          <div className={SUBSCRIPTION_ITEM_STYLE.leftBox}>
+            <p
+              className={SUBSCRIPTION_ITEM_STYLE.channel}
+            >{`채널: ${channel.channelName}`}</p>
+            <p
+              className={SUBSCRIPTION_ITEM_STYLE.platform}
+            >{`플랫폼: ${channel.platform}`}</p>
+          </div>
+          <div className={SUBSCRIPTION_ITEM_STYLE.rightBox}>
+            <span className={SUBSCRIPTION_ITEM_STYLE.dateText}>
+              {subscribedAt}
+            </span>
+            <button
+              onClick={() => handleClickButton()}
+              className={SUBSCRIPTION_ITEM_STYLE.deleteButton}
+            >
```

Comment:
```suggestion
            <button
              onClick={() => handleClick()}
              className={SUBSCRIPTION_ITEM_STYLE.deleteButton}
            >
```
물론 `handleClickButton` 함수가 더 명확하지만, 클릭 버튼이 이 컴포넌트에 여러개 있는게 아니라면
```javascript
on[EventName] <=> handle[EventName]
```
를 지켜주는게 더 좋지 않을까요?

Reply:
```suggestion
            <button
              onClick={() => setShowConfirmModal(true)}
              className={SUBSCRIPTION_ITEM_STYLE.deleteButton}
            >
```
함수 로직이 setShowConfirmModal(true)뿐이라서 별도 handle함수로 분리할 필요가 없을 것 같습니다.